dynamic linking doesn't work without Makefile modifications

When building gemma on my linux machine when using dynamic linking (i.e. when running `make FORCE_DYNAMIC=1`), I get the following error:

```
g++ -Wall -O3 -DWITH_LAPACK -m64 ./src/main.o ./src/param.o ./src/gemma.o ./src/io.o ./src/lm.o ./src/lmm.o ./src/vc.o ./src/mvlmm.o ./src/bslmm.o ./src/prdt.o ./src/mathfunc.o ./src/gzstream.o ./src/lapack.o -lgsl -lgslcblas -pthread -lz -llapack -o ./bin/gemma
/usr/bin/ld: ./src/lapack.o: undefined reference to symbol 'dgemm_'
/usr/bin/ld: note: 'dgemm_' is defined in DSO /usr/lib/libblas.so.3gf so try adding it to the linker command line
/usr/lib/libblas.so.3gf: could not read symbols: Invalid operation
collect2: ld returned 1 exit status
make: *** [bin/gemma] Error 1
```

This can be fixed for me by modifying the Makefile line setting the LIBS variable when FORCE_DYNAMIC is set to include `-lblas`

--- a/Makefile
+++ b/Makefile
@@ -32,7 +32,11 @@ CPP = g++
 
 CPPFLAGS = -Wall -O3
 
+ifdef FORCE_DYNAMIC
+LIBS = -lgsl -lgslcblas -lblas -pthread -lz
+else
 LIBS = -lgsl -lgslcblas -pthread -lz
+endif
 
 OUTPUT = $(BIN_DIR)/gemma
 

